### PR TITLE
Kls 1254 3 add video on demand cta to detail page

### PR DESCRIPTION
**Pull request by sebs-code**
This ticket adds a CTA column for Video on Demand for the UKEA Event pages.

TO TEST: GO to an event page in UKEA. In the admin panel for the event page, add a past recording (if it doesn't already have one). CTA Column should then appear as in the screenshots link on the original ticket.

### Workflow

- [X] Ticket exists in Jira https://uktrade.atlassian.net/browse/KLS-1254
- [X] Jira ticket has the correct status.
- [X] A clear/description pull request messaged added.

### Reviewing help

- [X] Explains how to test locally, including how to set up appropriate data
- [X] Includes screenshot(s) - ideally before and after, but at least after

### Merging

- [X] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: styles/great/_utils.scss [hunk 1]
```diff
@@ -849,3 +854,15 @@ img.richtext-image {
   max-height: 200px;
   overflow: auto;
 }
+
+.great-icon-box-container{
```
Comment (LuptonM) [inline styles/great/_utils.scss:858]: duplicate?

Comment (sebs-code) [reply styles/great/_utils.scss:858]: Good spot. Must have crept in during the rebase. Will remove.

File: export_academy/templates/export_academy/includes/event_details_main_content.html [hunk 2]
```diff
@@ -84,6 +84,32 @@ <h3 class="govuk-heading-s govuk-!-static-margin-bottom-2 govuk-!-static-margin-
                     </div>
                 {% endif %}
             </div>
+            <div class="govuk-grid-column-one-third-from-desktop govuk-!-static-padding-0">
+                {% if request.user.is_authenticated and ended and has_video and booked %}
```
Comment (timothyPatterson) [inline export_academy/templates/export_academy/includes/event_details_main_content.html:88]: Is this if statement necessary?

Comment (sebs-code) [reply export_academy/templates/export_academy/includes/event_details_main_content.html:88]: Yes - it's a very unique edge case that needs to be accounted for.

Comment (sebs-code) [reply export_academy/templates/export_academy/includes/event_details_main_content.html:88]: The Scenario is that if a user is logged in, and they event has ended, and it has a recording of said event, and I've booked on the course, don't show the column :-D.

Comment (timothyPatterson) [reply export_academy/templates/export_academy/includes/event_details_main_content.html:88]: OK lol - so basically someone can't join the course and then watch the video after?

**Review by timothyPatterson — APPROVED** [review]

**Pull request merged**